Lib/test/test_ast.py

```diff
@@ -260,11 +348,20 @@ def to_tuple(t):
   "()",
   # Combination
   "a.b.c.d(a.b[1:2])",
-
+  # Slice
+  "[5][1:]",
+  "[5][:1]",
+  "[5][::1]",
+  "[5][1:1:1]",
+  # IfExp
+  "foo() if x else bar()",
+  # JoinedStr and FormattedValue
+  "f'{a}'",
```

Comment:
Since `FormattedValue` cannot be used without a `JoinedStr`, so separate this snippets makes no sense.